app/_data/docs_nav_konnect.yml

```
@@ -115,6 +115,8 @@
       url: /mesh-manager/
     - text: Create a mesh
       url: /mesh-manager/service-mesh/
+    - text: RBAC
```

```suggestion
    - text: Role Based Access in MinK
```

Or something more explicit.